doc/tutorials/using_partition_mode_on_nuc.rst

```diff
@@ -213,17 +213,18 @@ Update Ubuntu GRUB to Boot Hypervisor and Load Kernel Image
 
    .. code-block:: console
 
-      menuentry 'ACRN hypervisor Logical Partition Scenario' --id ACRN_Logical_Partition --class ubuntu --class gnu-linux --class gnu --class os $menuentry_id_option 'gnulinux-simple-e23c76ae-b06d-4a6e-ad42-46b8eedfd7d3' {
+      menuentry 'ACRN hypervisor Partitioned Scenario' --id ACRN_Partitioned --class ubuntu --class gnu-linux --class gnu --class os $menuentry_id_option 'gnulinux-simple-e23c76ae-b06d-4a6e-ad42-46b8eedfd7d3' {
            recordfail
            load_video
            gfxmode $linux_gfx_mode
            insmod gzio
            insmod part_gpt
            insmod ext2
            search --no-floppy --fs-uuid --set 9bd58889-add7-410c-bdb7-1fbc2af9b0e1
-           echo 'Loading hypervisor logical partition scenario ...'
+           echo 'Loading hypervisor partitioned scenario ...'
            multiboot2  /boot/acrn.bin root=PARTUUID="e515916d-aac4-4439-aaa0-33231a9f4d83"
            module2 /boot/bzImage Linux_bzImage
+           module2 /boot/initrd_Image Ubuntu
```

Comment:
There is no tag at the moment for `ramdisk_mod` in the scenario XML. We need additional instructions for the user because he needs to add this before compiling ACRN.

Reply:
This parameter has been added to the scenario XML